be/src/exec/pipeline/scan_operator.cpp

```diff
@@ -115,6 +121,9 @@ void ScanOperator::set_finishing(RuntimeState* state) {
 
 StatusOr<vectorized::ChunkPtr> ScanOperator::pull_chunk(RuntimeState* state) {
     RETURN_IF_ERROR(_try_to_trigger_next_scan(state));
+    if (_workgroup != nullptr) {
+        _workgroup->decrease_chunk_num(1);
+    }
```

Comment:
pull_chunk may return nullptr, is it ok to call `decrease_chunk_num` in this situation?

Reply:
Yes, this shouldn't really be called decrease_chunk_num, it's meant to be a request to get chunk， maybe it should rename to ask_for_chunk ?, i would rename in next pr

Reply:
it just try to get, but not necessarily successful， but we need count the number of it